src/Components/LeftSideWeatherApp/CurrentDayWeatherForecast/CurrentDayWeatherForecast.js

```diff
@@ -1,23 +1,24 @@
 import './CurrentDayWeatherForecast.css';
                                   
 function CurrentDayWeatherForecast(props) {
-    const weekday = ["Sunday","Monday","Tuesday","Wednesday","Thursday","Friday","Saturday"];
+    const weekday = ['Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday'];
```

Comment:
put this outside the component so it does not get re-created each time the component is rendered. if the component re-renders 100 times then this array will be re-created 100 times given your computer more work to do since that array is not changing, put it outside the component so it only get's created once.